### DISCUSSION TITLE
What is the decode method here?

### DISCUSSION THREAD

**Discussion by AL-377** [outlines-dev/outlines#333] [Q&A] [answered]
Besides the logits modify ,is the decode method greedy search now?

**Comment by rlouf** [comment, marked as answer]
You can use both multinomial and greedy sampling.

**Reply by AL-377** [reply]
okok,I see the code position,thanks!
